i18n/locales/zh-Hant.json

```
@@ -68,6 +68,24 @@
   "manual_delivery_warning_label": "需等候作者發貨",
   "other_file_type": "其他",
   "price_free": "免費",
+  "pricing_page_continue_button": "繼續",
+  "pricing_page_feature_1": "AI 粵語及國語朗讀",
+  "pricing_page_feature_2": "5% 購書回贈，不設上限",
+  "pricing_page_feature_3": "優先使用新功能",
+  "pricing_page_feature_4": "真人回覆客服訊息",
+  "pricing_page_feature_5": "beta 特價，一經訂閱，永不漲價",
+  "pricing_page_liker_id_required": "需要 Liker ID",
+  "pricing_page_liker_id_required_description": "你需要有 Liker ID 才能訂閱。請聯絡客服獲取協助。",
```

We dont want to show Liker ID
```suggestion
  "pricing_page_liker_id_required": "需要註冊帳戶",
  "pricing_page_liker_id_required_description": "你需要註冊帳戶才能訂閱。請聯絡客服獲取協助。",
```

---

[💬 Update required field messages for registration](https://github.com/likecoin/3ook-com/pull/57/commits/0aa79e93625619a557292f3affdaf4b54323f175)